specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/disk.json

```diff
@@ -0,0 +1,3609 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DiskResourceProviderClient",
+    "description": "The Disk Resource Provider Client.",
+    "version": "2021-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/disks/{diskName}": {
+      "put": {
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_CreateOrUpdate",
+        "description": "Creates or updates a disk.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DiskNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "disk",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Disk"
+            },
+            "description": "Disk object supplied in the body of the Put disk operation."
+          }
+        ],
+        "responses": {
```

Comment:
ideally first commit needs to be baseline disk.json in the new folder helps reviewers identify the changes